src/components/accordion.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import { Collapse, Button, CardBody, Card } from 'reactstrap';
+import styled, { css } from "react-emotion";
+
+// CSS
+
+const StyledButton = styled(Button)`
```

Comment:
are you ok with this naming convention for styled components? My team in my old company used to do it this way and I didn't love it so wonder if you have other suggestions. It's not a big deal either way